__tests__/dashboard.test.js

```diff
@@ -16,6 +23,16 @@ test('Should redirect to no-accounts', async (t) => {
   await t.expect(title).ok();
 });
 
+test('Should show view only account in dashboard metrics', async (t) => {
+  await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
+  await addAccountPage.addViewOnly();
+
+  dashboardPage.waitPageLoaded();
+
+  FIXTURE_VIEW_ONLY_TOKENS.forEach((t) => dashboardPage.expectBalanceInBalanceList(t));
+  FIXTURE_VIEW_ONLY_TOKENS.forEach((t) => dashboardPage.expectTokenInTokenList(t));
+});
```

Comment:
Tricky.
I'm wondering if instead of separating tests by 'view', if we should reason in 'user flow'.
A series  of actions, through multiple views, that have the expected behaviour.

AddAccount Flow:
"When I add an account, I expect to be directed to the home page where the new account is in the list AND the balance of tokens are displayed in the  Balance  list"

Reply:
Hmm, perhaps that's a better idea